cmd/encryption-v1.go

```diff
@@ -438,6 +639,34 @@ func (o *ObjectInfo) IsEncrypted() bool {
 	return false
 }
 
+// IsEncrypted returns true if the object is marked as encrypted.
+func (li *ListPartsInfo) IsEncrypted() bool {
+	if _, ok := li.UserDefined[ServerSideEncryptionIV]; ok {
+		return true
+	}
+	if _, ok := li.UserDefined[ServerSideEncryptionSealAlgorithm]; ok {
+		return true
+	}
+	if _, ok := li.UserDefined[ServerSideEncryptionSealedKey]; ok {
+		return true
+	}
+	return false
+}
+
+func decryptedSize(encryptedSize int64) (int64, error) {
+	if encryptedSize == 0 {
+		return encryptedSize, nil
+	}
+	size := (encryptedSize / (sseDAREPayloadBlockSize + SSECustomerKeySize)) * sseDAREPayloadBlockSize
+	if mod := encryptedSize % (sseDAREPayloadBlockSize + SSECustomerKeySize); mod > 0 {
+		if mod < 33 {
```

Comment:
If you use a properly named constant such as `sseDAREPackageMetaSize`, you can rewrite the condition as `mod <= sseDAREPackageMetaSize` and the meaning becomes clear.